homeworks/maksim.sivachenko_MaxSiva/homework_5/obj.js

```diff
@@ -0,0 +1,9 @@
+const array = ['header', 'menu', 'menu_item', 'tabs',
+    'tab_item', 'menu', 'link', 'tabs', 'tab_item', 'menu',
+    'menu_item', 'menu', 'menu_item'];
+const result = {};
+
+array.forEach((a) => {
+    result[a] = result[a] + 1 || 1;
```

Comment:
looks nice, but... I would use `hasOwnProperty()` check here, or at least `for...in` check

in perspective, you never know what property name could be, it could be a not enumerable prototype method like `toString()` for example, such way you will reassign it